src/main/java/com/faforever/client/chat/ChatUserContextMenuController.java

```diff
@@ -309,4 +344,17 @@ public void onRemoveCustomColor() {
   public ContextMenu getRoot() {
     return chatUserContextMenuRoot;
   }
+
+  public void onKickGame() {
+    moderatorService.closePlayersGame(getPlayer().getId());
+  }
+
+  public void onKickLobby() {
+    moderatorService.closePlayersLobby(getPlayer().getId());
+  }
+
+
+  public void consumer(ActionEvent actionEvent) {
```

Comment:
What does this do? Java doc or rename maybe

Reply:
no, it consumes an action event, exactly what is says

Reply:
whats the effect?